hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -435,6 +465,9 @@ public static boolean retainOrUpdateRecord(
       case MISSING:
         returnValue = container.isMissing() && !container.isEmpty();
         break;
+      case EMPTY_MISSING:
```

Comment:
Pls explain the reason why we need to add this explicit `EMPTY_MISSING` switch case here because anyway we are deleting `EMPTY_MISSING` later at line [here]( https://github.com/apache/ozone/pull/7149/files#diff-5ecf43d8695ccad5a7f9d506546af1044b6034a2616787f8f881a276dc6f08fbR309)